SCF-Session-8-Assignment/src/question/Lion.java

```diff
@@ -0,0 +1,29 @@
+package question;
+
+/**
+ * This class is for Lion type animals
+ * @author Jugal
+ *  Dated 07/24/2019
+ */
+public class Lion extends Mammal {
+
+	/**
+	 * Constructor to initialize the object
+	 * 
+	 * @param name is the name of the animal
+	 * @param type is the type of the animal
+	 * @param category is the category of the animal
+	 */
+	public Lion(String name, String type, String category) {

```

Comment:
Declare some different properties of Lion class and initialize them in Lion class constructor and initialize Mammal and  Animal class property using super().

Reply:
Done